### ISSUE TITLE
[ios] [web] iOS and web Semantics do not support labeling EditTexts as disabled. 

### ISSUE THREAD

**Issue by hangyujin** [flutter/flutter#143337]
### Steps to reproduce

b/322345393

        TextField(
          enabled:false,
          decoration:InputDecoration(
            labelText: 'input label',
          )
      ),

### Expected results

read as 'dimmed' in ios

### Actual results

/

### Code sample

<details open><summary>Code sample</summary>

```dart
[Paste your code here]
```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
[Paste your output here]
```

</details>

**Comment by math1man** [comment]
FYI this also affects Web, not just iOS.

**Comment by math1man** [comment]
I'm fairly certain #143334 does not fix this issue on Web.

**Comment by hangyujin** [comment]
reopening this: https://github.com/flutter/flutter/pull/143334 fixed this on iOS but not on web

**Comment by jiahaog** [comment]
Updating to p0 to match internal priority.

**Comment by math1man** [comment]
Don't forget to reopen

**Comment by hangyujin** [comment]
I am trying to fix the g3 tests failure and reland https://github.com/flutter/flutter/pull/143334

**Comment by hangyujin** [comment]
Reassigning to web team since https://github.com/flutter/flutter/pull/143334 fixed this on iOS but not on web.

**Comment by math1man** [comment]
> They are just not announced as "dimmed" (or similar).

This is a major usability bug for accessibility users.

> AFAICT, this has always been the case, so it doesn't look like a recent regression.

I believe that's true, but that doesn't really impact the severity. If anything, it makes it more severe because we have no recourse to revert to an older version to fix the issue.

**Comment by yjbanov** [comment]
I'll try to fix this as part of https://github.com/flutter/flutter/issues/83809. Otherwise, when a text field is disabled it can still grab focus.
